src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.AuthenticationRequestDto;
+import mate.academy.spring.model.dto.response.AuthenticationResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import mate.academy.spring.service.dto.mapping.DtoResponseMapper;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final DtoResponseMapper<AuthenticationResponseDto, User> userDtoMapper;
```

Comment:
let's use UserResponseDto instead, also let's use UserRequestDto instead of AuthenticationRequestDto. Also remove these dtos